staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -33,6 +33,18 @@ var (
 
 	// timeNow stubbed out for testing
 	timeNow = time.Now
+
+	// lvlToZapLvl map log level to zap level
+	// follow guide line 
+	// https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
+	lvlToZap map[int]zapcore.Level = map[int]zapcore.Level{
```

Comment:
First klog verbosity starts at 0 for logs that should ALWAYS visible, not an Info level. 
By greping kubernetes repo I found verbosity up to 12.

Reply:
I don't think a map will work here.